src/main/groovy/life/qbic/datamodel/dtos/general/Person.groovy

```diff
@@ -27,6 +27,12 @@ abstract class Person {
    */
   final int id
 
+  /**
+   * Unique entity identifier (UUID)
+   * @since 2.22.0
+   */
+  final String referenceId
```

Comment:
I think we should store it as UUID here and only convert for the database

Reply:
why?

Reply:
Ah nvm those are only the DTOs